int/api/server.go

```diff
@@ -101,6 +102,10 @@ func initLocalAPI(localAPI *operations.ThyraServerAPI, app *fyne.App, manager *p
 	localAPI.WebsiteCreatorPrepareHandler = operations.WebsiteCreatorPrepareHandlerFunc(
 		websites.CreatePrepareForWebsiteHandler(app),
 	)
+
+	localAPI.CmdDeploySCHandler = operations.CmdDeploySCHandlerFunc(
+		deploysc.CreateCmdDeploySCHandler(),
+	)
```

Comment:
If you don't pass any argument your function should by passed as signature
```suggestion
	localAPI.CmdDeploySCHandler = operations.CmdDeploySCHandlerFunc(deploysc.CreateCmdDeploySCHandler)
```